src/tasks/tasks.repository.ts

```diff
@@ -27,8 +28,15 @@ export class TasksRepository extends Repository<Task> {
     return tasks;
   }
 
-  async createTask(createTaskDto: CreateTaskDto): Promise<Task> {
-    const task = this.create(createTaskDto);
+  async createTask(createTaskDto: CreateTaskDto, user: User): Promise<Task> {
+    const { title, description, status } = createTaskDto;
+
+    const task = this.create({
```

Comment:
Em có thể truyền hết vào như này được k?
```suggestion
    const task = this.create(createTaskDto)
    task.user = user;
    await this.save(task);
```

Reply:
Dạ em có thể truyền hết cái createTaskDto vào chung bằng việc sử dụng Spread Parameter ạ.

Reply:
ĐƯợc em nha, tùy ý em thôi. Cái này mục đích cho ngắn lại đỡ phải define từng cái attribute 1 như cái cũ